Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM ruby:2.4.2
+
+# Add apt repo for postgres 9.6 and install according to https://wiki.postgresql.org/wiki/Apt
```

Comment:
Why do we need the Postgres client installed? We've generally tried to minimize unnecessary dependencies in our production Docker images.